pkg/config/configuration.go

```diff
@@ -41,6 +41,8 @@ const (
 	ServiceInvocationStreaming Feature = "ServiceInvocationStreaming"
 	// Enables the app health check feature, allowing the use of the CLI flags
 	AppHealthCheck Feature = "AppHealthCheck"
+	// Enables the use of TLS below 1.2, which is insecure
+	AllowInsecureTLS Feature = "AllowInsecureTLS"
```

Comment:
Should we update the feature name to reflect it's only used for HTTP AppChannel? Otherwise it might be misleading.

Reply:
Completely Agree with the comment. I have used `AllowInsecureTLSForAppChannel`, does it work? I haven't used HTTP, as we would like to use same flag for grpc AppChannel as well.